server/api/index.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+
+const apiRouter = express.Router();
+
+//TODO: Fix issue that is making localhost:3000/api return with error saying "Cannot GET /api"
+
+// GET /api
+apiRouter.get('/', (req, res, next): void => {
+    try {
+        res.send('API is live');
+    } catch (e) {
+        next(e);
+    }
+})
+
+apiRouter.use((req, res): void => {
+    res.status(404)
+    .send({message: 'Invalid API endpoint'});
```

Comment:
nit: when chaining methods, I personally like when the second line is indented once to indicate it part of the expression above

```suggestion
    res.status(404)
        .send({message: 'Invalid API endpoint'});
```